src/tokens/solana.tokenlist.json

```diff
@@ -11915,7 +11915,32 @@
         "telegram": "https://t.me/kurobi_io",
         "twitter": "https://twitter.com/kurobi_io"
       }
-    } 
+    },
+    {
+      "chainId": 101,
+      "address": "FiCiuX9DetEE89PgRAU1hmoptnem8b1fkpEq8PGYTYkd",
+      "symbol": "MM",
+      "name": "Million Token",
```

Comment:
Can you please change name to "Million"? Thanks!

Reply:
Done!